hedyorg/hedy
# [FIX] Disable autoload and autosave of programs for slides

We used to only disable autoload, by checking an attribute of `"raw"` on the turtle canvas.

Change that into a JavaScript initialization property for clarity, and also use it to disable autosave.

**How to test**

Log in and go to the slides page, edit the code in the slides and click **Run**. 

Go your **My programs** page. Observe that the program is not there.
